### PR TITLE
PIM-10484: [SLA] Failed jobs shouldn't be filtered with 'In progress' filter in the process tracker

### PR DESCRIPTION
**Pull request by MadeWilson**
**Description (for Contributor and Core Developer)**

In the first fix we were fixing on corrupted job problematically after fetching them in the db. This solution wasn't sufficient since she didn't cover the case of a search. Here we directly fix the status value in the sql query to get both search and display of the job execution correct.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by StevenVAIDIE — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - PIM-10471: Do not generate 2 files when making a quick export of 1 type of products
 - PIM-10475: Fix option existence validation for numeric option codes
 - PIM-10483: Fix slow loading products when filtering by variants
+- PIM-10484: Fix job filter on status being incoherent with job interupted by deamon crash
```
Comment (StevenVAIDIE) [inline CHANGELOG.md:48]:
```suggestion
- PIM-10484: Fix job filter on status being incoherent with job interrupted by daemon crash
```

**Review by pierrejolly — APPROVED** [review]
IMO we should make a test to valid performance

File: src/Akeneo/Platform/Job/back/tests/Integration/Infrastructure/Query/SearchJobExecutionTest.php [hunk 2]
```diff
@@ -514,7 +512,7 @@ private function getExpectedJobExecutionRow(int $jobExecutionId): JobExecutionRo
                     'import',
                     new \DateTimeImmutable('2020-01-02T00:00:00+00:00'),
                     'peter',
-                    new JobExecutionHealthCheck(Status::fromLabel('IN_PROGRESS'), null, new \DateTimeImmutable('2020-01-02 01:00:00')),
+                    Status::fromLabel('IN_PROGRESS'),
```
Comment (ghoussard) [inline src/Akeneo/Platform/Job/back/tests/Integration/Infrastructure/Query/SearchJobExecutionTest.php:515]: Could you add a health check time in this fixture to simulate the case of the status is IN PROGRESS in database with an outdated health check time pls ?

**Review by ghoussard — APPROVED** [review]
GTM after testing this fix on a duplicated customer env (such as madeira)

**Pull request merged**